src/screens/lessonBase.js

```diff
@@ -29,6 +30,7 @@ const Lesson = ({route}) => {
 
   return (
     <Suspense fallback={loadingScreen()}>
+      <ProgressBar />
```

Comment:
is there a way to have it visible throughout the app instead of sticking it in every screen we create?

Reply:
@mark-yacoub  
There is by placing it with the navigator. Is there a preference at the top or the bottom ?

Reply:
Based on our latest discussion, we're keeping it as is to avoid it showing up in unwanted screens like About for now